server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/TranslatorOptionsUtils.java

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright (c) 2025 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.implicitDialects.cics;
+
+import com.google.common.collect.Sets;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import lombok.experimental.UtilityClass;
+import org.eclipse.lsp.cobol.common.dialects.DialectProcessingContext;
+import org.eclipse.lsp.cobol.common.model.Locality;
+import org.eclipse.lsp.cobol.common.model.tree.CompilerDirectiveNode;
+import org.eclipse.lsp.cobol.common.utils.StringUtils;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.Range;
+
+/** CICS translator options utils */
+@UtilityClass
+public class TranslatorOptionsUtils {
+  private static final int A_B_ARIA_LEN = 72 - 7;
+  private static final int ARIA_A = 7;
+  private static final Pattern CBL_LINE =
+      Pattern.compile("^(?<prefix>\\s*(CBL|PROCESS)\\s+)(?<cbl>.*)$", Pattern.CASE_INSENSITIVE);
+  private static final Pattern CBL_CICS =
+      Pattern.compile(
+          ".*(?<cics>CICS\\(\"?(?<args>([^()]|\\([^()]*\"?\\))+)\\))", Pattern.CASE_INSENSITIVE);
+  private static final Pattern CBL_XOPTS =
+      Pattern.compile(
+          ".*(?<xopts>XOPTS\\((?<args>([^()]|\\([^()]*\\))+)\\))", Pattern.CASE_INSENSITIVE);
+  public static final HashSet<Character> DELIMITERS = Sets.newHashSet(' ', ',');
+
+  /**
+   * Extract CICS translator options from CBL lines
+   *
+   * @param context is a dialect processing context
+   * @return List of CICS translator options
+   */
+  public List<CompilerDirectiveNode> extractCompilerDirectives(DialectProcessingContext context) {
```

Comment:
Everything else is already marked static manually anyway.
```suggestion
  public static List<CompilerDirectiveNode> extractCompilerDirectives(DialectProcessingContext context) {
```